tests/tests/payloads/issue_comment.json

```diff
@@ -0,0 +1,225 @@
+{
+  "action": "created",
```

Comment:
Is this correct? Action `created` in a payload for showcasing a comment?

Reply:
I copied-pasted events from the webhooks settings in github, I'm pretty sure they are right...

Reply:
Okey. Interesting, though.